vars/jobCommon.groovy

```diff
@@ -46,16 +46,20 @@ def Boolean onlyJenkinsRelease() {
 //  - file: debian package file
 //  - token: deb-drop token
 //  - repo: repository for uploading
+//  - keepVersions: Number of versions to keep
 def uploadPackage(Map config) {
     if (!fileExists(config.file)) {
         error "File ${config.file} doesn't exists, abort uploading to repo"
     }
+    if (!config.KeepVersions) {
+        config.keepVersions = 5
+    }
```

Comment:
It just came to my mind. It worths already to define `curl` variable once here and use it inside `config.emulate` closures, to not repeat ourselves.

Besides that, looks good

Reply:
I have to test, but it's here